src/main/java/com/store/entity/category/CategoryEntity.java

```diff
@@ -0,0 +1,57 @@
+package com.store.entity.category;
+
+import com.store.entity.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.Table;
+
+@javax.persistence.Entity
+@Table(name = "category")
+public class CategoryEntity extends Entity {
+
+    @Column(name = "NAME_CATEGORY")
+    private String nameCategory;
+
+    @Column(name = "DESCRIPTION")
+    private String description;
+
+    @Column(name = "SUBCATEGORY_ID")
+    private int subcategoryId;
```

Comment:
It is better to use "parentCategoryId" (as 1 category can have 1 parent and many childs)